vector/src/main/java/im/vector/app/features/raw/wellknown/ElementWellKnownExt.kt

```diff
@@ -16,11 +16,14 @@
 
 package im.vector.app.features.raw.wellknown
 
+import org.matrix.android.sdk.api.auth.data.SessionParams
 import org.matrix.android.sdk.api.extensions.tryOrNull
 import org.matrix.android.sdk.api.raw.RawService
 
-suspend fun RawService.getElementWellknown(userId: String): ElementWellKnown? {
-    return tryOrNull { getWellknown(userId) }
+suspend fun RawService.getElementWellknown(sessionParams: SessionParams): ElementWellKnown? {
+    // By default we use the domain of the userId to retrieve the .well-known data
+    val domain = sessionParams.userId.substringAfter(":")
```

Comment:
Should we have an extension method for this substringAfter(":") as we use it at several places

Reply:
Done: added `fun String.getDomain(): String` in MatrixPattern